sw/airborne/modules/computer_vision/cv/resize.h

```diff
@@ -36,13 +36,13 @@ inline void resize_uyuv(struct img_struct *input, struct img_struct *output, int
       *dest++ = *source++; // U
       *dest++ = *source++; // Y
       // now skip 3 pixels
-      if (pixelskip) { source += (pixelskip + 1) * 2; }
+      source += (pixelskip + 1) * 2;
```

Comment:
I would like to keep my version, otherwise it doesn't work with downsample=1 (which makes a copy of the image)